### DISCUSSION TITLE
Rgb gone while laptop is charged

### DISCUSSION THREAD

**Discussion by kisz09** [seerge/g-helper#1497] [Q&A]
Hi, 
i downloaded the 0.129 version but suddenly my rgb not working.
tried to go back to 0.126 again but still rgb is not working.

i shutdown the laptop and remove everything that plug including the charging cable.
open it after 10mins. and the rgb is now  working again.
so i just leave it unplugged until 80% and charged it again.
suddenly the rgb is not working again after i charged it. 
P.S. still  on 0.126 version.

anyone knows how to solve this? it happens when i upgrade to 0.129 version.

Laptop specs:
Asus Rog Scar 16 (rtx 4090)
windows 11
32gb ram

thanks

**Comment by seerge** [comment]
@kisz09 i don't know what do you mean under "not working" specifically (does it turn off ? or it doesn't follow selected mode?) but most probably you have leftovers from AC running, and I would recommend to stop them :)

**Comment by kisz09** [comment]
hello, what i mean by not working is that the keyboard rgb is not glowing anymore, no light.
i just messes up everything when i upgraded from 0.126 to 0.129

**Reply by seerge** [reply]
@kisz09 there are no changes for RGB in 0.129 :)

**Comment by kisz09** [comment]
thanks. that good to know.
